corehq/apps/app_manager/templates/app_manager/profile.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <profile version="{{ app.version }}"
-         update="{{ profile_url }}"
+         update="{{ profile_url }}&amp;profile={{ build_profile }}"
```

Comment:
`profile_url` should always end in `?latest=true` by my reading of the code. because of https://github.com/dimagi/commcare-hq/blob/lang-profiles/corehq/apps/app_manager/models.py#L4844-L4847 and https://github.com/dimagi/commcare-hq/blob/lang-profiles/corehq/apps/app_manager/models.py#L4220-L4230